core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/performance/clientside/ClientSidePerformanceCollector.java

```diff
@@ -22,7 +22,7 @@
 import com.cognifide.aet.job.api.collector.WebCommunicationWrapper;
 import com.cognifide.aet.job.api.exceptions.ParametersException;
 import com.cognifide.aet.job.api.exceptions.ProcessingException;
-import com.cognifide.aet.job.common.utils.JsRuntimeWrapper;
+import com.cognifide.aet.job.common.utils.javaScript.JsRuntimeWrapper;
```

Comment:
Please use lowercase only in package names

Reply:
Sorry! Fixed!